### PR TITLE
Fix linter warnings

### LINKED ISSUE #240: Fix the Linter warnings
Fix the linter issues, so that the next years students do not get warnings when opening the project for the first time, as we did with Xamarin studios. 

Document the public members to fix the linter warnings.

### PR DESCRIPTION
**Pull request by LukasRH**
Closes #240

### CODE REVIEW

**Comment by Sworgon** [comment]
@aau-giraf/sw608f19 and @aau-giraf/sw609f19 you have been assigned this. Here's your checklist. I marked @Tricky12321, @jenspetur94, and @TrineHolmager as the reviewer, but you might change it as you like.
Please use the following checklist. Feel free to check it off as you go along.
**Code Design**
- [ ] Is the code in the right place? (Both in terms of folder structure and class structure)
- [ ] Could this code, to the best of your knowledge, have reused existing code?
- [ ] Is the code over-engineered? Examples of over-engineering: Implemented behavior for future problems
- [ ] Does it introduce functionality that is not necessary for solving the problem?

**Code Readability and Maintainability**
- [ ] Are names meaningful and self-documenting?
- [ ] Is it understandable, by reading the code, what it does?
- [ ] Is the code simple and readable? i.e. it contains no “hack”, or obscure solution.
- [ ] Is it understandable, by reading the tests, what they do?
- [ ] Do the tests cover sensible cases? Both happy paths and exception paths?
- [ ] Do the tests cover the full functionality? i.e., will the tests fail if some of the requested functionality is missing?
- [ ] Are error messages in the code understandable?
- [ ] Are confusing sections of code covered by comments?

**Functionality**
For this part, open the app, and test:
- [ ] Are the features that the code claims to implement, actually implemented?
- [ ] Does it look like the code has bugs?
- [ ] Are all new screens reachable through in-app navigation?

**Points for Discussion and Reflection**
- [ ] Are UI messages, that the user might meet, actually helpful?
- [ ] Are there errors that are bad practice? Like, hard-coding a test database connection?
- [ ] Is there well written code in this? Remember to tell the developer!

File: lib/blocs/add_activity_bloc.dart [hunk 1]
```diff
@@ -2,18 +2,14 @@ import 'package:weekplanner/blocs/bloc_base.dart';
 import 'package:rxdart/rxdart.dart';
 import 'package:flutter/material.dart';
 
-
-
-
-///Bloc used for adding activites to a citizen's day
+///Bloc used for adding activities to a citizen's day
```
Comment (swuf) [inline lib/blocs/add_activity_bloc.dart]:
```suggestion
/// Bloc used for adding activities to a citizen's day
```

**Review by swuf — CHANGES_REQUESTED** [review]

**Review by swuf — APPROVED** [review]
No warnings 🎈😃

**Review by Tricky12321 — APPROVED** [review]

**Review by andpaacoke — APPROVED** [review]
No UI changes so approved from PO 👌

**Pull request merged**